### PR TITLE
Ethereum V2 Reserve Rates Update

### PR DESCRIPTION
**Pull request by yonikesel**
Discussion: https://governance.aave.com/t/arfc-chaos-labs-incremental-reserve-factor-updates-aave-v2-ethereum/13766

### CODE REVIEW

File: src/AaveV2EthereumRatesUpdates_20230627/AaveV2EthereumRatesUpdates_20230627Test.t.sol [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      AaveV2Ethereum.POOL
+    );
+
+    diffReports(
+      'preTestAaveV2EthereumRatesUpdates_20230627',
+      'postTestAaveV2EthereumRatesUpdates_20230627'
+    );
+
+    address[] memory assetsChanged = new address[](18);
+    assetsChanged[0] = AaveV2EthereumAssets.DAI_UNDERLYING;
```
Comment (ronlv10) [inline src/AaveV2EthereumRatesUpdates_20230627/AaveV2EthereumRatesUpdates_20230627Test.t.sol:40]: can you do instead 

    address[] memory assetsChanged = [DAI_UNDERLYING, FRAX_UNDERLYING ...] ?

File: src/AaveV2EthereumRatesUpdates_20230627/ETHEREUM_V2_PARAMETER_UPDATES-20230627.md
Comment (ronlv10) [inline src/AaveV2EthereumRatesUpdates_20230627/ETHEREUM_V2_PARAMETER_UPDATES-20230627.md:1]: image
I think DAI is wrong ? should be 10 according to https://governance.aave.com/t/arfc-chaos-labs-incremental-reserve-factor-updates-aave-v2-ethereum/13766

File: diffs/preTestAaveV2EthereumRatesUpdates_20230627_postTestAaveV2EthereumRatesUpdates_20230627.md
Comment (ronlv10) [inline diffs/preTestAaveV2EthereumRatesUpdates_20230627_postTestAaveV2EthereumRatesUpdates_20230627.md:1]: [here](https://governance.aave.com/t/arfc-chaos-labs-incremental-reserve-factor-updates-aave-v2-ethereum/13766) the value for TUSD is different, is that ok ?

File: src/AaveV2EthereumRatesUpdates_20230627/AaveV2EthereumRatesUpdates_20230627Test.t.sol [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot(
+      'preTestAaveV2EthereumRatesUpdates_20230627',
+      AaveV2Ethereum.POOL
+    );
+
+    // 2. create payload
+    proposalPayload = new AaveV2EthereumRatesUpdates_20230627();
+
+    // 3. execute payload
+
```
Comment (ronlv10) [inline src/AaveV2EthereumRatesUpdates_20230627/AaveV2EthereumRatesUpdates_20230627Test.t.sol]: remove space

**Review by ronlv10 — APPROVED** [review]

File: src/AaveV2EthereumRatesUpdates_20230627/AaveV2EthereumRatesUpdates_20230627Test.t.sol (hunk 1 above)
Comment (yonikesel) [reply src/AaveV2EthereumRatesUpdates_20230627/AaveV2EthereumRatesUpdates_20230627Test.t.sol:40]: no

**Pull request closed without merging**